### ISSUE TITLE
Housekeeping traceback

### ISSUE THREAD

**Issue by ryanmerolle** [netbox-community/netbox-docker#784]
### Current Behavior

If I clone the repo fresh, and start docker-compose (using the example override included in the repo), I noticed that the housekeeping container has some traceback error messages.

Again this is an deployment without any changes to the current repo.  This does not stop the spin up of the containers/services overall, so I doubt anyone has realized this.

### Expected Behavior

No traceback

### Docker Compose Version

Docker Compose version v2.6.0

### Docker Version

```text
Client: Docker Engine - Community
 Cloud integration: v1.0.25
 Version:           20.10.16
 API version:       1.41
 Go version:        go1.17.10
 Git commit:        aa7e414
 Built:             Thu May 12 09:17:39 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Desktop
 Engine:
  Version:          20.10.16
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.10
  Git commit:       f756502
  Built:            Thu May 12 09:15:42 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.4
  GitCommit:        212e8b6fa2f44b9c21b2798135fc6fb7c53efc16
 runc:
  Version:          1.1.1
  GitCommit:        v1.1.1-0-g52de29d
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### The git Revision

61a3afbb3b83b294bf87d6d34e78b318f03476da

### The git Status

```text
On branch release
Your branch is up to date with 'origin/release'.

nothing to commit, working tree clean
```

### Startup Command

docker-compose up

### NetBox Logs

```text
netbox-docker-netbox-housekeeping-1  | Interval set to 86400 seconds
netbox-docker-netbox-housekeeping-1  | Thu Jun 30 12:02:57 UTC 2022
netbox-docker-netbox-housekeeping-1  | Skipping config initialization (database unavailable)
netbox-docker-netbox-housekeeping-1  | Traceback (most recent call last):
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
netbox-docker-netbox-housekeeping-1  |     return self.cursor.execute(sql, params)
netbox-docker-netbox-housekeeping-1  | psycopg2.errors.UndefinedTable: relation "django_session" does not exist
netbox-docker-netbox-housekeeping-1  | LINE 1: DELETE FROM "django_session" WHERE "django_session"."expire_...
netbox-docker-netbox-housekeeping-1  |                     ^
netbox-docker-netbox-housekeeping-1  |
netbox-docker-netbox-housekeeping-1  |
netbox-docker-netbox-housekeeping-1  | The above exception was the direct cause of the following exception:
netbox-docker-netbox-housekeeping-1  |
netbox-docker-netbox-housekeeping-1  | Traceback (most recent call last):
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/netbox/manage.py", line 10, in <module>
netbox-docker-netbox-housekeeping-1  | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox-docker-netbox-housekeeping-1  | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox-docker-netbox-housekeeping-1  | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox-docker-netbox-housekeeping-1  | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox-docker-netbox-housekeeping-1  | [*] Clearing expired authentication sessions
netbox-docker-netbox-housekeeping-1  |     execute_from_command_line(sys.argv)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
netbox-docker-netbox-housekeeping-1  |     utility.execute()
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 440, in execute
netbox-docker-netbox-housekeeping-1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 414, in run_from_argv
netbox-docker-netbox-housekeeping-1  |     self.execute(*args, **cmd_options)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 460, in execute
netbox-docker-netbox-housekeeping-1  |     output = self.handle(*args, **options)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/netbox/extras/management/commands/housekeeping.py", line 30, in handle
netbox-docker-netbox-housekeeping-1  |     engine.SessionStore.clear_expired()
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/contrib/sessions/backends/db.py", line 110, in clear_expired
netbox-docker-netbox-housekeeping-1  |     cls.get_model_class().objects.filter(expire_date__lt=timezone.now()).delete()
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/query.py", line 827, in delete
netbox-docker-netbox-housekeeping-1  |     deleted, _rows_count = collector.delete()
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/deletion.py", line 462, in delete
netbox-docker-netbox-housekeeping-1  |     count = qs._raw_delete(using=self.using)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/query.py", line 843, in _raw_delete
netbox-docker-netbox-housekeeping-1  |     cursor = query.get_compiler(using).execute_sql(CURSOR)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1361, in execute_sql
netbox-docker-netbox-housekeeping-1  |     cursor.execute(sql, params)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
netbox-docker-netbox-housekeeping-1  |     return self._execute_with_wrappers(
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
netbox-docker-netbox-housekeeping-1  |     return executor(sql, params, many, context)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
netbox-docker-netbox-housekeeping-1  |     return self.cursor.execute(sql, params)
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
netbox-docker-netbox-housekeeping-1  |     raise dj_exc_value.with_traceback(traceback) from exc_value
netbox-docker-netbox-housekeeping-1  |   File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
netbox-docker-netbox-housekeeping-1  |     return self.cursor.execute(sql, params)
netbox-docker-netbox-housekeeping-1  | django.db.utils.ProgrammingError: relation "django_session" does not exist
netbox-docker-netbox-housekeeping-1  | LINE 1: DELETE FROM "django_session" WHERE "django_session"."expire_...
netbox-docker-netbox-housekeeping-1  |                     ^
netbox-docker-netbox-housekeeping-1  |
```

### Content of docker-compose.override.yml

```yaml
version: '3.4'
services:
  netbox:
    ports:
      - '8000:8080'
```

**Comment by tobiasge** [comment]
This happens only on the first start, when the database is not yet initialised. The housekeeping container tries to clean old sessions but the session table doesn't exists yet.

I don't think this a "real" problem, but it could be solved by doing the sleep in the script before the housekeeping call.

**Comment by ryanmerolle** [comment]
I totally agree.  I was just pointing it out.

**Comment by tobiasge** [comment]
@ryanmerolle You can try out these changes: https://github.com/tobiasge/netbox-docker/commit/c8e385b690b664017f46ec0b5ead9422de1b752c

I've added health checks and optimised the `depends_on` setting. Now the `housekeeping` and `worker` start after the web service is healthy. Which is after all migrations have been applied and nginx ist responding.

**Comment by ryanmerolle** [comment]
Apologies that I lost track of this.  @cimnine may I take care of the a pull request for this given @tobiasge is away?

**Comment by tobiasge** [comment]
Closed with #866
